src/com/pentalog/pentastagiu/homework2/shop/Customer.java

```diff
@@ -0,0 +1,43 @@
+package com.pentalog.pentastagiu.homework2.shop;
+
+public class Customer {
+    private int idCustomer;
+    private String name;
+    private Membership userMembership;
+
+    public Customer(int idCustomer, String name, Membership userMembership) {
+        this.idCustomer = idCustomer;
```

Comment:
o implementare mai potrivita pentru id(identificator pentru un obiect) ar fi generarea lui printr-un mecanism mai degraba decat setarea de catre cineva(astfel te poti asigura ca e unic; cu ce am invatat poti face asta prin folosirea unui field static counter care e incrementat de fiecare data cand e create un obiect nou iar in constrructor, nu vom mai da id-ul ca parametru, ci il vom initializa cu valoarea counterului)